### DISCUSSION TITLE
feature request: User Provided Static Image on Pause

### DISCUSSION THREAD

**Discussion by lemmy04** [Motion-Project/motion#1473] [General]
1. version [x.y.z, hash, other]: 
4.0
2. installed as a package or compiled from sources [deb, rpm, git, other]: 
package
3. standalone or part of third party [motion, MotionEyeOS, other]: 
standalone
4. video stream source [V4L (card or USB), net cam (mjpeg, rtsp, other), mmal]: 
raspberry pi cam 2.0
5. hardware [x86, ARM, other]: 
raspberry pi nano w - arm
6. operating system [Linux (which), FreeBSD, other]: 
debian 9.1

Is there a way to switch motion between serving the camera stream, and servince one static image that has been saved to disk?

Right now I just stop the camera software when it gets dark outside, and start it again at sunrise, I'd like to feed a static image instead, can't find anything in the settings...

**Comment by tosiara** [comment]
Why do you need to feed a static image? To stop motion detection?

**Reply by lemmy04** [reply]
I have a camera that is off during the night, so instead of the camera feed I would want a static image during those hours.
